armory/instrument/config.py

```diff
@@ -253,6 +257,31 @@ def _task_metric(
     elif name == "word_error_rate":
         final = metrics.get("total_wer")
         final_suffix = "total_word_error_rate"
+    elif name == "per_class_mean_accuracy":
+        metric = metrics.get("identity_unzip")
+        func = metrics.get("per_class_mean_accuracy")
+
+        def final(x):
+            return func(*metrics.task.identity_zip(x))
+
+        final_suffix = name
+    elif name == "confusion_matrix":
+        metric = metrics.get("identity_unzip")
+        func = metrics.get("confusion_matrix")
+
+        def final(x):
+            return func(*metrics.task.identity_zip(x))
+
+        final_suffix = name
+    elif name == "precision_and_recall":
+        metric = metrics.get("identity_unzip")
+        func = metrics.get("precision_and_recall")
+
+        def final(x):
+            return func(*metrics.task.identity_zip(x))
+
+        final_suffix = name
+
```

Comment:
Can you combine these into a single block? E.g., 
```python
    elif name in ("per_class_mean_accuracy", "confusion_matrix", "precision_and_recall"):
        metric = metrics.get("identity_unzip")
        func = metrics.get(name)

        def final(x):
            return func(*metrics.task.identity_zip(x))

        final_suffix = name
```

Reply:
Yep that was on my todo list